frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -315,18 +314,30 @@ class MidCircuitMeasure(HybridOp):
 
     binder = qmeasure_p.bind
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.postselect = self.in_classical_tracers[-2]
-        self.reset = self.in_classical_tracers[-1]
+    # pylint: disable=too-many-arguments
+    def __init__(
+        self,
+        in_classical_tracers,
+        out_classical_tracers,
+        regions: List[HybridOpRegion],
+        reset: bool = None,
+        postselect: int = None,
+    ):
+        HybridOp.__init__(self, in_classical_tracers, out_classical_tracers, regions)
+        self.reset = reset
+        self.postselect = postselect
+        # self.mv = self.out_classical_tracers[0]
 
-    def trace_quantum(self, ctx, device, trace, qrp) -> QRegPromise:
-        op = self
-        wire = op.in_classical_tracers[0]
+    # pylint: disable=too-many-arguments
+    def trace_quantum(self, ctx, device, trace, qrp, postselect_mode=None) -> QRegPromise:
+        wire = self.in_classical_tracers[0]
```

Comment:
Maybe we can assign it in the init as well?

Reply:
We would need a way for `catalyst.measure` to know the `postselect_mode` for that.

Reply:
For the wire? I just meant parsing out the wire on init from the classical tracers, mainly for "nicecess" but very minor.

Reply:
Oh sorry I got confused. Thought you meant the `postselect_mode`.